api4/group.go

```diff
@@ -84,6 +84,10 @@ func (api *API) InitGroup() {
 	api.BaseRoutes.Groups.Handle("/{group_id:[A-Za-z0-9]+}",
 		api.APISessionRequired(requireLicense(deleteGroup))).Methods("DELETE")
 
+	// GET /api/v4/groups/:group_id
+	api.BaseRoutes.Groups.Handle("/{group_id:[A-Za-z0-9]+}/undelete",
```

Comment:
We're using `/restore` for channels and teams (and `/active` for users). Do you think we should use `/restore` for groups too?